.github/workflows/ci.yml

```diff
@@ -437,7 +438,6 @@ jobs:
         # FIXME (yao): sometimes there is cache but we still miss argilla in the env; so we add make install-ci again
         make install-ci
         sudo apt-get update && sudo apt-get install --yes poppler-utils libreoffice
-        make install-pandoc
```

Comment:
same story, but this time `make install-ci` is being called above
https://github.com/Unstructured-IO/unstructured/pull/2593/files#r1507798453